src/test/java/com/example/pfairplayservice/jpa/repository/MemberRepositoryIntegrationTest.java

```diff
@@ -53,4 +61,179 @@ public void whenFindById_thenReturnMemberEntity() {
         assertThat(found.get().getUid(), is(equalTo(memberEntity.getUid())));
     }
 
+    @Test
+    public void whenFindByMemberTeamTid_thenReturnMemberEntity() {
+        MemberEntity memberEntity = MemberEntity
+                .builder()
+                .id("ryoo")
+                .password("testpw")
+                .address("testad")
+                .birthday(new Date())
+                .joinDate(new Date())
+                .level(1)
+                .preferPosition(1)
+                .phoneNumber("010")
+                .recentLoginDate(new Date())
+                .phoneNumberDisclosureOption(1)
+                .name("ryoo2")
+                .build();
+
+        TeamEntity teamEntity = TeamEntity
+                .builder()
+                .teamName("teamnametest")
+                .teamLeadMember(memberEntity)
+                .activityAreaAddress("seoul")
+                .registrationDate(new Date())
+                .foundDate(new Date())
+                .build();
```

Comment:
It is hard to read and understand test case.
make generating method as private method with random and reuse it.

For example,
```
private TeamEntity generateTeamEntity(memberEntity){
         String name = RandomStringUtils.random(8);
        TeamEntity teamEntity = TeamEntity
                .builder()
                .teamName(name)
                .teamLeadMember(memberEntity)
                .activityAreaAddress("seoul")
                .registrationDate(new Date())
                .foundDate(new Date())
                .build();
}
```

Reply:
I think so too. I will